sql-parser/src/main/java/io/crate/sql/tree/Insert.java

```diff
@@ -31,11 +31,16 @@
     protected final Table<T> table;
     private final DuplicateKeyContext<T> duplicateKeyContext;
     protected final List<String> columns;
+    protected final List<SelectItem> returning;
 
-    Insert(Table<T> table, List<String> columns, DuplicateKeyContext<T> duplicateKeyContext) {
+    Insert(Table<T> table,
```

Comment:
Could `Insert` and `InsertFromSubquery` merged into one class ? `Insert seems not be used anywhere.

Reply:
Yes we should do that. Could you follow up on that against master before continueing with this here?